completions/configure

```diff
@@ -20,7 +20,7 @@ _configure()
             ;;
     esac
 
-    if $split || [[ $cur != -* ]]; then
+    if "$split" || [[ $cur != -* ]]; then
```

Comment:
As I mentioned above, this is contamination by my mistake. These changes are moved to #918 but I forgot to drop it from the branch.

Reply:
> Also, it seems there are a lot of `$split && return`s without quoting left after this PR, was that intentional?

`$split` is intentionally separated to #918. Because it is related to the interface change of `_comp_initialize`, I think we need special discussions for `$split` set by `_comp_initialize` (as suggested in https://github.com/scop/bash-completion/pull/918#issuecomment-1518503463).